monitoring/dashboard.py

```diff
@@ -525,6 +557,173 @@ def top10_errors_by_bucket(title, code):
     title="5xx : Top10 by Bucket", code='~"5.."'
 )
 
+
+quotaHealth = TimeSeries(
```

Comment:
you might want to put a max at `100%` as it shows 200% in your screenshot.

In the legend you could put the latest value so we don't have to hover to see the tooltip

Reply:
Unfortunately it's not yet available in the python lib we use, I'll see if I can do something about it.